addons/mail/static/tests/composer/composer_tests.js

```diff
@@ -850,21 +850,6 @@ QUnit.test("remove an uploading attachment", async (assert) => {
     assert.containsNone($, ".o-mail-Composer .o-mail-AttachmentCard");
 });
 
-QUnit.test("Show a thread name in the recipient status text.", async (assert) => {
-    const pyEnv = await startServer();
-    const partnerId = pyEnv["res.partner"].create({ name: "test name", email: "test@odoo.com" });
-    pyEnv["mail.followers"].create({
-        is_active: true,
-        partner_id: partnerId,
-        res_id: partnerId,
-        res_model: "res.partner",
-    });
-    const { openFormView } = await start();
-    await openFormView("res.partner", partnerId);
-    await click("button:contains(Send message)");
-    assert.containsOnce($, ".o-mail-Chatter:contains(To: test)");
-    assert.containsOnce($, 'span[title="test@odoo.com"]');
-});
```

Comment:
Test is still relevant. It just needs to be adapted so that followers is subscribed to `mt_comment` subtype, in order to be listed as a recipients

Reply:
Adapting this test would make it redundant with this other test that already assert the text: 
https://github.com/odoo-dev/odoo/blob/saas-16.4-mail-filter-recipients-list-did/addons/mail/static/tests/composer/composer_tests.js#L854

This is why it's removed.

Reply:
Oh I see, then good riddance :)